src/mono/mono/mini/mini-posix.c

```diff
@@ -517,48 +518,11 @@ static volatile gint32 sampling_thread_running;
 
 #ifdef HOST_DARWIN
 
-static clock_serv_t sampling_clock_service;
+static clock_serv_t sampling_clock;
```

Comment:
Where is this set?  It used to be set in `clock_init()` below from a call to `host_get_clock_service()`, but now it's kept at zero.

Reply:
It is set in `mono_clock_init` in mono-time.c